src/style/inspector.scss

```diff
@@ -286,3 +286,21 @@ output[name="registry-status"] {
   display: block;
   font-weight: normal;
 }
+
+.ngrok-play {
+  border-color: green;
```

Comment:
For consistency I'd try to use `$green` (and `$red` below) unless it is particularly ugly. This should use the palette from bulma.